src/diff/component.js

```diff
@@ -195,7 +196,7 @@ export function renderComponent(
 		commitQueue.push(c);
 	}
 
-	c._force = false;
+	internal._mode &= ~FORCE_UPDATE;
```

Comment:
Ooo, could we add `FORCE_UPDATE` to the `RESET_MODE` constant and then just reset at the end of `patch` and `mount` when we reset other flags? Could save us a byte or two to not have this additional line of code

Reply:
Neat, you're right that's better 👍